### DISCUSSION TITLE
What causes the calculation speed to be different for each time period?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#857] [Q&A] [answered]
Recently I have a [AcousticTest](https://github.com/buzhiqiuyuebai/WaterAcousticTest) in water, the model shows as the picture below, I found that the computation speed varies for each time period, for example, the computation time spent in the 0 s to 5 s is much larger than that in the 10 s to 15 s period. What is the theory behind this, please? this is [outputlog](https://github.com/buzhiqiuyuebai/WaterAcousticTest/blob/main/outputlog) about output Information
![image](https://github.com/SeisSol/SeisSol/assets/114474894/59f109d7-817c-41df-93c8-4da433e27d5c)

**Comment by Thomas-Ulrich** [comment, marked as answer]
There is no reason that could explain a variation in the performance with time IMO, probably hardware related.
